recipes/graphviz/run_test.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
Given graphviz neither depends on Python, pygraphviz nor dask, I'm inclined to suggest that the testing is the wrong way around - this is an excellent test, but think it should live in the dask recipe.

What do you think @ccordoba12?